packages/@aws-cdk/aws-s3/lib/bucket.ts

```diff
@@ -199,6 +199,13 @@ export interface BucketImportProps {
    * @default Inferred from bucket name
    */
   bucketDomainName?: string;
+
+  /**
+   * The website URL of the bucket (if static web hosting is enabled).
```

Comment:
I'm not sure how useful this description is, I tried a couple of different variations but couldn't come up with anything better than this.

Suggestions welcome!